### ISSUE TITLE
Question: Passing a SSH key as a variable to the Playbook

### ISSUE THREAD

**Issue by Herotel-JohannK** [ansible-semaphore/semaphore#2302]
### Question

Hi All

So I need to pass a private key (That is saved in Semaphore) as a variable to the playbook in order to be used in the playbook to connect to another device as an example.

How would I be able to pass this and how would my playbook look?

### Related to

Ansible (task execution)

**Comment by e36Alex** [comment]
i got the same question!

**Comment by skarczjh** [comment]
While not a direct answer to your question, have you thought about using Ansible Vault to encrypt the key, and then calling your variable in your play? SemaphoreUI passes the Ansible Vault password just fine. This would also allow you to have auditable version control of the private key.

**Comment by e36Alex** [comment]
> While not a direct answer to your question, have you thought about using Ansible Vault to encrypt the key, and then calling your variable in your play? SemaphoreUI passes the Ansible Vault password just fine. This would also allow you to have auditable version control of the private key.

No, i didn't think about it until now. But i think it would be easier for me, if it would work with the semaphore keystore.
If it definitely don't work, its a way to think about.
